docs/getting-started/proofs-and-verification/proofs.md

```diff
@@ -44,4 +47,54 @@ message GetDataProofResponse {
 message MPTrieProofElement {
   repeated bytes hashes = 1;
 }
-```
\ No newline at end of file
+```
+
+### Proofs
+
+#### Ledger connectivity proof
+Proof generation:
+- Zero step – user provided with receipt (block header) from another user who claim that some block is part of ledger
+- First step – user requests header of last block in ledger, without discovering what block he wants to validate
+  - User can store genesis block apriori or request it as well
+- Second step – user requests the shortest path in skip list from block he validates to genesis block
+- The third step – user requests the shortest path from last block (first step) to block he validates
+- The forth step – user validates all apriori known headers - genesis, block, last - in both paths and concatenate both paths, result is block proof
```

Comment:
user -> Alice

shortest path in the skip list from the block she validates to the genesis block
shortest path from the last block of the ledger to the block she validates

apriori -> a-priori

both paths; the result is the block existence proof